dle/search/static/search/es_search.js

```diff
@@ -1,10 +1,10 @@
-    /* global instantsearch algoliasearch */
+/* global instantsearch algoliasearch */
 
     // Basic auth with username/password is not supported - bug: see https://github.com/searchkit/searchkit/issues/1235
     var globalSearchTerm = '';
     const sk = new Searchkit({
         connection: {
-          host: "http://localhost:8000/api/v1/searchkit", // TODO remove hardcoding for deployment
+            host: ELASTIC_HOST, // Set by the Django template in which this file is embedded
```

Comment:
Good to see this example. I was wondering how we might be able to get python variables into the js.

Reply:
Yeah - there are some security considerations but I'm using a filter to mitigate. See: https://adamj.eu/tech/2020/02/18/safely-including-data-for-javascript-in-a-django-template/